sdk/core/core/inc/az_contract.h

```diff
@@ -10,13 +10,18 @@
 
 #include <_az_cfg_prefix.h>
 
-#define AZ_CONTRACT_ARG_NOT_NULL(arg) \
+#define AZ_CONTRACT(condition, error) \
   do { \
-    if ((arg) == NULL) { \
+    if (!(condition)) { \
```

Comment:
this should do the thing we discussed where it calls a "panic" function, and returns its result (assuming it doesn't crash)

Reply:
Yes, we have a placeholder for this function now. So it's an improvement ;-)